src/main/scala/edu/washington/cs/seguard/js/JSFlowGraph.scala

```diff
@@ -265,3 +290,38 @@ object JSFlowGraph {
     }
   }
 }
+//          // DefUse based analysis
```

Comment:
Why are you commenting out a large block of code?

Reply:
Sorry I forgot to remove it. I copy the codes as reference when merging with your codes. It was removed now.